### PR TITLE
PWN-3401 - Add error state for history screen

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket 

https://p2pvalidator.atlassian.net/browse/PWN-3401

## Description of Work 

Add error state view for history screen

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -47,6 +47,8 @@
     <color name="backgroundBannerSecondary">#D7D1FF</color>
     <color name="colorBackgroundNotification">#404040</color>
 
+    <color name="secondaryGray">#6F7D8D</color>
```
Comment (1sakov) [inline app/src/main/res/values/colors.xml:50]: I am not sure if we have such color in our design system

**Comment by 1sakov** [comment]
@davran1001 minor comments. Please use the existing colors

**Review by 1sakov — APPROVED** [review]

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/res/values/colors.xml:50]: i think new feature is not correctly added according our design system
we need to aks Gena to add all colors to design system or change designs according to our design system =(

Comment (1sakov) [reply app/src/main/res/values/colors.xml:50]: The current history design doesn't fit the general design. 
Secondary gray color is `textIconSecondary`

Please work with designers and ask to change it. 
The history design does have wrong font as well

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**